src/components/components/event/event.js

```diff
@@ -0,0 +1,49 @@
+import "./event.scss";
+
+import React from "react";
+import { shortDate } from "./../../../utils/dates";
+
+import { Calendar as CalendarIcon } from "react-feather";
+
+// Wrapper for events.
+export default class Event extends React.PureComponent {
+  render() {
+    let classes = ["event"];
+    const now = Math.round(new Date().getTime() / 1000);
+    const closingDate = Math.max(
+      this.props.event.startDate,
```

Comment:
startDate and endDate are used a lot. Maybe adding a `const { startDate, endDate } = this.props.event` would simplify the codebase a lot